sniffer/main.c

```diff
@@ -45,7 +47,7 @@
 /**
  * @brief   Stack for the raw dump thread
  */
-static char rawdmp_stack[THREAD_STACKSIZE_MAIN];
+static char rawdmp_stack[THREAD_STACKSIZE_SMALL];
```

Comment:
This change may need verification on some more platforms though (AVR, msp430, mips)

Reply:
`z1` is an msp430 platform, so this I already tested. I'm not sure XBee works with the sniffer, but we can maybe try with an OpenLabs for the AVR-based board. Same goes probably for MIPS, though SPI is just in the works.

Reply:
The `z1` would even be happy with `THREAD_STACKSIZE_TINY` btw ;-)